nextcord/ui/view.py

```diff
@@ -505,16 +494,13 @@ def __init__(self, state: ConnectionState):
         self._synced_message_views: Dict[int, View] = {}
         self._state: ConnectionState = state
 
-    @property
-    def persistent_views(self) -> Sequence[View]:
```

Comment:
isnt this a breaking change?

Reply:
Nope, the view store is private to users.